tests/python/contrib/test_hexagon/test_thread_pool.py

```diff
@@ -66,8 +66,8 @@ def benchmark_func(mod, name, args, hexagon_session):
     return evaluator(a, b, c, n).mean
 
 
-@requires_hexagon_toolchain
-def test_speedup(hexagon_session, capsys):
+@tvm.testing.requires_hexagon
+def test_speedup(hexagon_session: Session, capsys):
     if hexagon_session is None:
```

Comment:
Since `hexagon_session` will be non-null whenever `ANDROID_SERIAL_NUMBER` is set, and `tvm.testing.requires_hexagon` requires that `ANDROID_SERIAL_NUMBER` is set, should we also use this to remove the `if hexagon_session is None` checks?

Reply:
make sense, removed the if condition.